src/components/slider/slider.component.ts

```diff
@@ -453,14 +455,39 @@ export class SliderComponent implements OnInit, AfterViewInit, DoCheck {
 
     }
 
-    private updateOrder(thumb: SliderThumb): void {
+    private updateOrder(thumb?: SliderThumb): void {
```

Comment:
You've packed another function inside this one, controlled by whether the parameter is provided. These are called from different places in the component, so there is no need for the function to figure out which of two completely distinct code paths to take. It's cleaner and more maintainable to have functions do one thing. An easy way to spot this pattern is when the function body is almost entirely inside an if/else statement.